tests/unit/test_entrypoints_api_handlers.py

```diff
@@ -183,9 +183,11 @@ def test_delete_diff_404(mock_delete_dataset):
     assert "<dataset> does not exist." in str(e.value)
 
 
+@patch('rdf_differ.entrypoints.api.handlers.get_diff')
```

Comment:
Do we still have to mock/patch or this is only for the 500 Error. Last time, I remember we decided to minimise this practice.